Bugzilla/CGI.pm

```diff
@@ -513,6 +513,16 @@ sub header {
     );
     $self->{_header_done} = 1;
 
+    my @fonts = (
+        "skins/standard/fonts/FiraMono-Regular.woff2?v=3.202",
+        "skins/standard/fonts/FiraSans-Bold.woff2?v=4.203",
+        "skins/standard/fonts/FiraSans-Italic.woff2?v=4.203",
+        "skins/standard/fonts/FiraSans-Regular.woff2?v=4.203",
+        "skins/standard/fonts/FiraSans-SemiBold.woff2?v=4.203",
+        "skins/standard/fonts/MaterialIcons-Regular.woff2",
+    );
+    $headers{'-link'} = join(", ", map { sprintf('</static/v%s/%s>; rel="preload"; as="font";', BUGZILLA->VERSION, $_) } @fonts);
```

Comment:
Woops, typo! Not sure why it isn't an error...
But the class name is Bugzilla. Only the method (VERSION) is all-caps.

Also, remove the final ; after "font"

Reply:
Fixed